lib/midcom/helper/datamanager2/widget/tinymce.php

```diff
@@ -173,13 +173,19 @@ private function _add_initscript($mode = 'exact')
 },
 IMG;
         }
-
+        $guid = $this->_type->storage->object->guid;
         // Compute the final script:
         $script = <<<EOT
 tinyMCE.init({
 {$config}
 {$this->local_config}
 mode : "{$mode}",
+plugins: "image imagetools",
+imagetools_toolbar: "editimage imageoptions",
+images_upload_url: "/__ais/imagepopup/upload/image/{$guid}/",
+images_upload_base_path: "/",
+images_upload_credentials: true,
+imagetools_cors_hosts: ['openpsa', 'localhost'],
```

Comment:
in general, can you move the image configuration down to where the individual configs are defined (or move it into a dedicated function)? If they stay where they are right now, they cannot by overridden by local configuration